### ISSUE TITLE
EdisonException: CORLEX not found in class path at resources/CORLEX/corelex_nouns

### ISSUE THREAD

**Issue by danyaljj** [IllinoisCogComp/illinois-cogcomp-nlp#105]
I am getting file not exception when using [`CorlexFeatureExtractor`](https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/blob/master/edison/src/main/java/edu/illinois/cs/cogcomp/edison/features/factory/CorlexFeatureExtractor.java):

```
[error] Exception in thread "main" edu.illinois.cs.cogcomp.edison.utilities.EdisonException: CORLEX not found in class path at resources/CORLEX/corelex_nouns
[error] 	at edu.illinois.cs.cogcomp.edison.features.factory.CorlexFeatureExtractor.loadData(CorlexFeatureExtractor.java:55)
[error] 	at edu.illinois.cs.cogcomp.edison.features.factory.CorlexFeatureExtractor.getWordFeatures(CorlexFeatureExtractor.java:94)
[error] 	at edu.illinois.cs.cogcomp.edison.features.WordFeatureExtractor.getFeatures(WordFeatureExtractor.java:79)
[error] 	at org.allenai.ari.solvers.termselector.ExpandedDataModel$$anonfun$56.apply(ExpandedDataModel.scala:274)
[error] 	at org.allenai.ari.solvers.termselector.ExpandedDataModel$$anonfun$56.apply(ExpandedDataModel.scala:273)
[error] 	at edu.illinois.cs.cogcomp.saul.datamodel.property.features.discrete.DiscreteProperty$$anon$2.discreteValue(DiscreteProperty.scala:67)
[error] 	at edu.illinois.cs.cogcomp.saul.datamodel.property.features.discrete.DiscreteProperty$$anon$2.featureValue(DiscreteProperty.scala:60)
[error] 	at edu.illinois.cs.cogcomp.saul.datamodel.property.features.discrete.DiscreteProperty.addToFeatureVector(DiscreteProperty.scala:77)
[error] 	at edu.illinois.cs.cogcomp.saul.datamodel.property.CombinedDiscreteProperty$$anon$1$$anonfun$classify$1.apply(CombinedDiscreteProperty.scala:32)
[error] 	at edu.illinois.cs.cogcomp.saul.datamodel.property.CombinedDiscreteProperty$$anon$1$$anonfun$classify$1.apply(CombinedDiscreteProperty.scala:32)
[error] 	at scala.collection.immutable.List.foreach(List.scala:381)
[error] 	at edu.illinois.cs.cogcomp.saul.datamodel.property.CombinedDiscreteProperty$$anon$1.classify(CombinedDiscreteProperty.scala:32)
[error] 	at edu.illinois.cs.cogcomp.lbjava.learn.Learner.getExampleArray(Learner.java:717)
[error] 	at edu.illinois.cs.cogcomp.lbjava.learn.Learner.getExampleArray(Learner.java:654)
[error] 	at edu.illinois.cs.cogcomp.lbjava.learn.Learner.learn(Learner.java:305)
[error] 	at edu.illinois.cs.cogcomp.saul.classifier.Learnable$$anonfun$learn$1$$anonfun$apply$mcVI$sp$1.apply(Learnable.scala:226)
[error] 	at edu.illinois.cs.cogcomp.saul.classifier.Learnable$$anonfun$learn$1$$anonfun$apply$mcVI$sp$1.apply(Learnable.scala:226)
[error] 	at scala.collection.mutable.ResizableArray$class.foreach(ResizableArray.scala:59)
[error] 	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
[error] 	at edu.illinois.cs.cogcomp.saul.classifier.Learnable$$anonfun$learn$1.apply$mcVI$sp(Learnable.scala:226)
[error] 	at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:166)
[error] 	at edu.illinois.cs.cogcomp.saul.classifier.Learnable.learn(Learnable.scala:221)
[error] 	at edu.illinois.cs.cogcomp.saul.classifier.Learnable.learn(Learnable.scala:199)
[error] 	at org.allenai.ari.solvers.termselector.EssentialTermsApp.trainAndTestLearner(EssentialTermsApp.scala:112)
[error] 	at org.allenai.ari.solvers.termselector.EssentialTermsApp.trainAndTestExpandedLearner(EssentialTermsApp.scala:41)
[error] 	at org.allenai.ari.solvers.termselector.EssentialTermsApp$.main(EssentialTermsApp.scala:368)
[error] 	at org.allenai.ari.solvers.termselector.EssentialTermsApp.main(EssentialTermsApp.scala)
```

Here is my code: 
```scala 
  val corlexFeatureExtractor = property(essentialTermTokens) { x: Constituent =>
    CorlexFeatureExtractor.instance.getFeatures(x).asScala.mkString
  }
```
I can see that the data exists in the resources, but it's not just being loaded.

**Comment by danyaljj** [comment]
And if we fix this we should add U test for it.

**Comment by christos-c** [comment]
Fixed it with #107, but no easy way to test it since the binarised TAs don't contain the Corelex features.
